src/Controller/AddOnOptionsController.py

```diff
@@ -261,33 +261,38 @@ def onCustomContextMenuRequestedVolume(self, pos):
                 self.table_volume.setItem(c, 0, QTableWidgetItem(new_data[0]))
                 self.table_volume.setItem(c, 1, QTableWidgetItem(new_data[1]))
 
-    # This option is extra and not fully supported yet, TO DO in the future
-    # # ROI from IsoDoses
-    # @QtCore.pyqtSlot(QtCore.QPoint)
-    # def on_customContextMenuRequested_Roi(self, pos):
-    #     it = self.table_roi.itemAt(pos)
-    #     if it is None:
-    #         return
-    #     c = it.row()
-    #     item_range = QtWidgets.QTableWidgetSelectionRange(
-    #         c, 0, c, self.table_roi.columnCount() - 1)
-    #     self.table_roi.setRangeSelected(item_range, True)
-    #
-    #     menu = QtWidgets.QMenu()
-    #     modify_row_action = menu.addAction("Modify")
-    #     menu.addSeparator()
-    #     delete_row_action = menu.addAction("Delete")
-    #     action = menu.exec_(self.table_roi.viewport().mapToGlobal(pos))
-    #     if action == delete_row_action:
-    #         self.table_roi.removeRow(c)
-    #     if action == modify_row_action:
-    #         dialog = Dialog_Dose(self.table_roi.item(
-    #             c, 0).text(), self.table_roi.item(c, 2).text())
-    #         if dialog.exec():
-    #             new_data = dialog.getInputs()
-    #             self.table_roi.setItem(c, 0, QTableWidgetItem(new_data[0]))
-    #             self.table_roi.setItem(c, 1, QTableWidgetItem(new_data[1]))
-    #             self.table_roi.setItem(c, 2, QTableWidgetItem(new_data[2]))
+    # ROI from IsoDoses
+    @Slot(QtCore.QPoint)
+    def onCustomContextMenuRequested_Roi(self, pos):
```

Comment:
I'm a bit confused, is this supposed to be `snake_case` or `camelCase`? :joy: better modify it for consistency?

Reply:
Change has been made and added to the PR.